packages/help-center/src/components/help-center-article.tsx

```diff
@@ -36,13 +33,6 @@ export const HelpCenterArticle = ( { navigateToRoute }: { navigateToRoute: strin
 		}
 	}, [ postUrl, post ] );
 
-	// Handle redirecting to new routes
-	useEffect( () => {
-		if ( navigateToRoute && navigateToRoute !== pathname + search + hash ) {
-			navigate( navigateToRoute );
-		}
-	}, [ navigateToRoute, pathname, search, hash, navigate ] );
-
```

Comment:
Does re-does what we're doing [here](https://github.com/Automattic/wp-calypso/pull/93196/files#diff-d2074a1d5192e133f78462249474fc4870d9962bd0c23bd765b44e2f63d451ffR87).

Reply:
Yeah, same logic.